generator/templates/scss/scss/app.scss

```diff
@@ -0,0 +1,31 @@
+@import "~bulma/sass/utilities/initial-variables";

```

Comment:
It is not a good idea to bring some buefy code to this project because we are duplicating code and we will have to keep it updated as well.